### DISCUSSION TITLE
Coupled run with 1/4 deg ocean/ice

### DISCUSSION THREAD

**Discussion by cecilehannay** [NCAR/amwg_dev#426] [Discussion about coupled runs]
Gustavo started a coupled run with 1/4 deg ocean/ice based on "54" configuration. This run has been only running for a 1 year thus far. 

After the co-chair meeting, we started "58" in which we revert to `cldfrc2m_do_avg_aist_algs = .false.` as this might have a negative impact ENSO. 
Gustavo and I are wondering whether we should wait for a few days to see the difference between "54" and "58" and see if we should base the  1/4 deg ocean/ice run  on "54" or on on "58"

Thoughts?
   
![Screen Shot 2023-11-15 at 10 52 42 AM](https://github.com/NCAR/amwg_dev/assets/9723220/1f632f4d-8e05-45ff-b9d8-c9930c235f1a)

@gustavo-marques, @PeterHjortLauritzen, @adamrher, @JulioTBacmeister, @dlawrenncar

**Comment by PeterHjortLauritzen** [comment]
I think it makes sense to wait until we have ENSO diags for "58"

**Reply by cecilehannay** [reply]
@gustavo-marques: let's hold off on the 1/4 deg until "58" is long enough.

**Reply by gustavo-marques** [reply]
Sounds good, thanks!
